modules/boss/bz/config.py

```diff
@@ -18,6 +18,11 @@ def parse_bz_config(config):
         bzs[bz]['server'] = config.get(bz, 'bugzilla_server')
         bzs[bz]['user'] = config.get(bz, 'bugzilla_user')
         bzs[bz]['password'] = config.get(bz, 'bugzilla_pwd')
+        if config.has_option(bz, "use_http_auth"):
+            bzs[bz]['use_http_auth'] = config.getboolean(bz, "use_http_auth")
+        else:
+            bzs[bz]['use_http_auth'] = False
```

Comment:
Teeny comment: Elsewhere (django settings.py) we seem to use the idiom of setting default and changing if present. ie:
```
       bzs[bz]['use_http_auth'] = False
       if config.has_option(bz, "use_http_auth"):
             bzs[bz]['use_http_auth'] = config.getboolean(bz, "use_http_auth")

```